src/gui/controls/volumecontrol.cpp

```diff
@@ -138,6 +143,15 @@ void VolumeControlPrivate::changeDisplay(VolumeControl::Options options, bool in
         updateDisplay(m_settings->value<Settings::Core::OutputVolume>());
         updateButtonStyle();
     }
+
+    if(options & VolumeControl::Tooltip) {
+        QObject::connect(m_volumeSlider, &LogSlider::sliderMoved, [this](int value) { updateToolTip(value); });
+        QObject::connect(m_volumeSlider, &LogSlider::sliderReleased, [this]() {
+            if(m_toolTip) {
+                m_toolTip->deleteLater();
+            }
+        });
+    }
```

Comment:
You'll want to pass a context object as the 3rd argument for the connections. Using the same sender `m_volumeSlider` will do.